templates/fungible-asset-template/frontend/pages/CreateFungibleAsset.tsx

```diff
@@ -145,7 +153,7 @@ export function CreateFungibleAsset() {
               <Input
                 type="number"
                 onChange={(e) => {
-                  setCostToMint(e.target.value);
+                  setMintFeePerFA(parseInt(e.target.value));
```

Comment:
why this change? label says "cost to mint" but event sets mint fee per fa.

we should keep the same naming

Reply:
nft template UI says `MintFeePerNFT`

Reply:
then lates change the UI label here also? https://github.com/aptos-labs/create-aptos-dapp/pull/68/files#diff-01771c38a69a7a89669fad355a6d0b769a3de007ed1560828051d77be0bd023eR152

Reply:
done